pkg/engine/update.go

```diff
@@ -132,6 +132,9 @@ type UpdateOptions struct {
 	// true if the plan should refresh before executing.
 	Refresh bool
 
+	// true if the plan should use program based destroy.
```

Comment:
```suggestion
	// true if the plan should run the program as part of destroy.
```
Just a thought; happy either way